dll/shellext/zipfldr/zipfldr.spec

```diff
@@ -2,4 +2,4 @@
 @ stdcall -private DllGetClassObject(ptr ptr ptr)
 @ stdcall -private DllRegisterServer()
 @ stdcall -private DllUnregisterServer()
-@ stdcall RouteTheCall(ptr ptr wstr long)
\ No newline at end of file
+@ stdcall RouteTheCallW(ptr ptr wstr long)
```

Comment:
```suggestion
@ stdcall RouteTheCallW(ptr ptr wstr long)

```

Reply:
In my usual defense, it was already like that when I got there and no warning about it missing.